### ISSUE TITLE
A delivery note can be issued from an existing order

### ISSUE THREAD

**Issue by myriamboure** [openfoodfoundation/wishlist#144]
## Description
**- As a:** enterprise manager

**- On page:** somewhere in menu admin/orders (to be defined when working on UX)

**- I want to be able to do:** THIS NEED TO BE SPLIT IN MULTIPLE STORIES, starting with that placeholder for now.
- create a delivery note for a given order by entering quantity delivered for the corresponding customer
- create delivery notes by batch by selecting multiple orders at a time
- create delivery notes with an entry by customer (for hubs who prepare baskets per customers one by one) an by products (for hubs who take a product and split it into whole baskets before dealing with next product)  
- generate a PDF for the delivery not and attach it to order line

## Acceptance criteria
- Enterprise users can print accurate delivery notes from existing orders

**Comment by drummer83** [comment]
@RachL 
Looks like this one ended up in Ready to go by mistake, didn't it?

**Comment by RachL** [comment]
yes it got connected by mistake to an issue that was in this column :+1:
